project/src/main.cpp

```diff
@@ -1 +1,22 @@
-// Write code here
\ No newline at end of file
+#include<iostream>
+using namespace std;
+    class company{
+        public : 
+        string name;
+        void input( string n){
+            name = n ;
+            
+        }
+        void display(){
+            cout << name;
+        }
+    };
+
+int main(){
```

Comment:
Remove Main function, we will add the main function in main.cpp

Reply:
this is main.cpp

Reply:
Hmm then above classes are in the wrong file, the Company class should be in Company.cpp !!